app/controllers/voters_controller.rb

```diff
@@ -64,44 +65,46 @@ def create
     end
 
     def update
-      if !admin_logged_in?
-        redirect_to "/"
+      # could allow for (timing based) Voter enumeration
```

Comment:
I don't quite understand what timing-based voter enumeration means in this context